suplemon/modules/keymap.py

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8
+
+import os
+
+from suplemon.suplemon_module import Module
+
+
+class Keymap(Module):
```

Comment:
Sometimes it's okay to not be DRY (e.g. when we can't predict the end result/are exploring a new feature). In this case, we more/less know that the API is going to look like Sublime's format so abstractions are welcome =)